metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFactory.java

```diff
@@ -75,17 +79,17 @@ public HiveConnectorFactory(final String name, final Map<String, String> configu
     private void init() {
```

Comment:
I'm not sure I understand the purpose of an `init()` method that is only called from the constructor anyway. Why not just do the logic in the constructor? I've seen @ajoymajumdar do this as well so he might have a reason